scripts/setup-jest.ts

```diff
@@ -0,0 +1,76 @@
+class EventEmitter {
```

Comment:
This is a setup function shared by all jest tests. This is where the mock vscode module is created.